src/main/java/gregtech/api/unification/material/materials/FirstDegreeMaterials.java

```diff
@@ -0,0 +1,1227 @@
+                .color(0xE6EBEB)
+                .components(Barium, 1, Sulfur, 1, Oxygen, 4)
+                .build();
+
+        Alunite = new Material.Builder(388, "alunite")
+                .dust()
+                .color(0xE1B441).iconSet(METALLIC)
+                .components(Potassium, 1, Aluminium, 3, Silicon, 2, Hydrogen, 6, Oxygen, 14)
+                .build();
+
+        Dolomite = new Material.Builder(389, "dolomite")
+                .dust(1)
+                .color(0xE1CDCD).iconSet(FLINT)
+                .components(Calcium, 1, Magnesium, 1, Carbon, 2, Oxygen, 6)
+                .build();
+
+        Wollastonite = new Material.Builder(390, "wollastonite")
+                .dust()
+                .color(0xF0F0F0)
+                .components(Calcium, 1, Silicon, 1, Oxygen, 3)
+                .build();
+
+        Kaolinite = new Material.Builder(391, "kaolinite")
+                .dust()
+                .color(0xF3EBEB)
+                .components(Aluminium, 2, Silicon, 2, Hydrogen, 4, Oxygen, 9)
+                .build();
+
+        Talc = new Material.Builder(392, "talc")
+                .dust().ore()
+                .color(0x5AB45A).iconSet(FINE)
+                .components(Magnesium, 3, Silicon, 4, Hydrogen, 2, Oxygen, 12)
+                .build();
+
+        Soapstone = new Material.Builder(393, "soapstone")
+                .dust(1).ore(3, 1)
+                .color(0x5F915F)
+                .components(Magnesium, 3, Silicon, 4, Hydrogen, 2, Oxygen, 12)
+                .build();
+
+        Kyanite = new Material.Builder(394, "kyanite")
+                .dust()
+                .color(0x6E6EFA).iconSet(FLINT)
+                .components(Aluminium, 2, Silicon, 1, Oxygen, 5)
+                .build();
+
+        IronMagnetic = new Material.Builder(395, "iron_magnetic")
+                .ingot()
+                .color(0xC8C8C8).iconSet(MAGNETIC)
+                .flags(EXT2_METAL, MORTAR_GRINDABLE)
+                .components(Iron, 1)
+                .ingotSmeltInto(Iron)
+                .arcSmeltInto(Iron)
+                .macerateInto(Iron)
+                .build();
+        Iron.getProperty(PropertyKey.INGOT).setMagneticMaterial(IronMagnetic);
+
+        TungstenCarbide = new Material.Builder(396, "tungsten_carbide")
+                .ingot().fluid()
+                .color(0x330066).iconSet(METALLIC)
+                .flags(EXT2_METAL)
+                .components(Tungsten, 1, Carbon, 1)
+                .toolStats(12.0f, 4.0f, 1280)
+                .fluidPipeProperties(7568, 125, true)
+                .blastTemp(2460)
+                .build();
+
+        CarbonDioxide = new Material.Builder(397, "carbon_dioxide")
+                .fluid(Material.FluidType.GAS)
+                .color(0xA9D0F5)
+                .components(Carbon, 1, Oxygen, 2)
+                .build();
+
+        TitaniumTetrachloride = new Material.Builder(398, "titanium_tetrachloride")
+                .fluidTemp(2220)
```

Comment:
should this have a `.fluid()` builder call?

Reply:
For readability, probably. Though functionally, it does not